examples/CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ blt_add_executable(
   DEPENDS_ON ${example_depends})
 list(APPEND umpire_examples allocator)
 
+blt_add_executable(
```

Comment:
This needs to be guarded - only build if `ENABLE_ALLOCATION_BACKTRACE` is On.

Reply:
But it shows both the case were ALLOCATION_BACKTRACE is optionally on and the case when exceptions are generated containing backtrace information.

Reply:
Okay, I understand.